packages/zip-it-and-ship-it/src/runtimes/node/bundlers/zisi/resolve.ts

```diff
@@ -3,7 +3,7 @@ import { version as nodeVersion } from 'process'
 
 import { findUp } from 'find-up'
 import { pathExists } from 'path-exists'
-// @ts-expect-error types are wrong
+// @ts-expect-error doesnt export async
```

Comment:
mhm are the types not correct? Maybe we should update `@types/node`?

Reply:
This is an NPM package, not a node stdlib one. let's handle this separately